android/src/main/java/io/bluedot/BluedotPointSdkModule.java

```diff
@@ -133,110 +352,20 @@ private Notification createNotification(String channelId,String channelName,Stri
             return notification.build();
         }
     }
-    @Override public void onBlueDotPointServiceStartedSuccess() {
-        serviceManager.subscribeForApplicationNotification(this);
-    }
-
-    @Override public void onBlueDotPointServiceStop() {
-        if (logOutCallback != null) {
-            logOutCallback.invoke("Success");
-
-            // Reset callbacks.
-            // React Native Callbacks can be invoked only once. More info: https://reactnative.dev/docs/native-modules-android.html#callbacks
-            logOutCallback = null;
-        }
-        serviceManager.unsubscribeForApplicationNotification(this);
-    }
-
-    @Override public void onBlueDotPointServiceError(BDError bdError) {
-        WritableMap error = new WritableNativeMap();
-        error.putString("error",bdError.getReason());
-        if (bdError instanceof LocationServiceNotEnabledError) {
-            sendEvent(reactContext,"startRequiringUserInterventionForLocationServices",error);
-            serviceManager.stopPointService();
-        } else if (bdError instanceof BluetoothNotEnabledError) {
-            sendEvent(reactContext,"startRequiringUserInterventionForBluetooth",error);
-            serviceManager.stopPointService();
-        }
-
-    }
-
-    @Override public void onRuleUpdate(List<ZoneInfo> list) {
-        WritableArray zoneList = new WritableNativeArray();
-        if(list != null) {
-            for (int i = 0; i < list.size(); i++) {
-                WritableMap zone = new WritableNativeMap();
-                zone.putString("name",list.get(i).getZoneName());
-                zone.putString("ID",list.get(i).getZoneId());
-                zoneList.pushMap(zone);
-            }
-        }
-        WritableMap map = new WritableNativeMap();
-        map.putArray("zoneInfos",zoneList);
-        sendEvent(reactContext, "zoneInfoUpdate",map);
-    }
 
     @Override
     public void onCheckIntoFence(FenceInfo fenceInfo, ZoneInfo zoneInfo, LocationInfo locationInfo,
```

Comment:
I think we can remove this

Reply:
onCheckIntoFence is part of callbacks declared in ApplicationNotificationListener, BluedotPointSdkModule implements ApplicationNotificationListener class, so this cannot be removed otherwise it will lead compile issues

Reply:
That makes sense 👍